prac_10/wiki.py

```diff
@@ -0,0 +1,17 @@
+import wikipedia
+
+
+search = input('Enter page title / search phrase: ')
+while search != '':
+    try:
+        page = wikipedia.page(search)
+        print(page.title)
+        print(page.summary)
+        print(page.url)
+    except wikipedia.exceptions.DisambiguationError as e:
+        print('Please enter a more specific search phrase, such as:')
+        for option in e.options[:5]:
```

Comment:
This `:5` feels a bit "magic numbery". A constant or comment would help us understand it better.